src/main/java/org/sagebionetworks/web/client/widget/table/v2/results/TableQueryResultWidget.java

```diff
@@ -306,6 +306,11 @@ private void setQueryResultsAndSort(QueryResultBundle bundle, List<SortItem> sor
 	 * @return
 	 */
 	public boolean isQueryResultEditable() {
```

Comment:
The result of this method is passed into TableEntityWidget.queryExecutionFinished. Line 461 triggers visibility of the edit button:

 https://github.com/Sage-Bionetworks/SynapseWebClient/blob/a8d2a3386ddc871e6440975c711d14eb6d823211/src/main/java/org/sagebionetworks/web/client/widget/table/v2/TableEntityWidget.java#L457-L483